src/lib/modules/blockchain_process/httpProxyOverride.js

```diff
@@ -0,0 +1,136 @@
+/* global require */
```

Comment:
It would have been cool to create new file in TS ;)
next time.

Reply:
Yeah forgot. Also, this file is kind of a hack, because it patches the function in `http-proxy` with a new function from a PR that was never merged.